config/networking-wildcardcert.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: networking-wildcardcert
```

Comment:
I would prefer naming it as `networking-ns` as I think we may want to have this controller for handling namespace related works instead of only namespace-level certificate. WDYT?
/cc @mattmoor @tcnghia

Reply:
probably `networking-ns-cert`